src/content/docs/en/guides/content-collections.mdx

```diff
@@ -388,9 +388,9 @@ The `slug` property is specific to content collections, and will not be availabl
 You can use this to filter by any content criteria you like. For example, you can filter by properties like `draft` to prevent any draft blog posts from publishing to your blog:
 
 ```js
-// Example: Filter content entries with `draft: true`
+// Example: Filter out content entries with `draft: true`
 import { getCollection } from 'astro:content';
-const draftBlogEntries = await getCollection('blog', ({ data }) => {
+const blogEntries = await getCollection('blog', ({ data }) => {
```

Comment:
```suggestion
const publishedBlogEntries = await getCollection('blog', ({ data }) => {
```
Ok, I'm suggesting it @colmanhumphrey! Docs are for teaching/learning, so let's be super explicit!  :smile:

Reply:
I think it makes sense — it's easy to understand and when people use this in their actual code, they know they can just change it